libmscore/figuredbass.cpp

```diff
@@ -1186,7 +1186,7 @@ qDebug("FiguredBass: duration indicator end line not implemented");
 void FiguredBass::draw(QPainter* painter) const
       {
       // if not printing, draw duration line(s)
-      if (!score()->printing() ) {
+      if (!score()->printing() && !score()->showUnprintable()) {
```

Comment:
Hi. I tested the change but the way I understood the issue was wrong. I just follow the bug description, which was not correct: "Shouldn't the gray line that appears above figured bass numbers **disappear when one selects View/show unprintable**?". I understood the gray lines should disappear when an user checks the Show unprintable checkbox, instead of appearing.
Anyway, I am glad that the issue was solved.